swap_meet/decor.py

```diff
@@ -1,2 +1,12 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Decor(Item):
+    def __init__(self, id = None, condition=0, width=0, length=0):
+        super().__init__(id)
+        self.condition = condition
```

Comment:
Same comment as in `Clothing` class, we should pass `condition` to the parent class constructor since the logic already exists to set the attribute in `Item` class and we want to avoid duplicating logic. 

If you refactor this logic in `Clothing`, `Decor`, `Electronics`, you'd remove 3 unnecessary lines of code from the codebase.

```suggestion
        super().__init__(id, condition)
```